packages/authentication/test/acceptance/basic-auth.ts

```diff
@@ -127,24 +127,24 @@ describe('Basic Authentication', () => {
     class MySequence implements SequenceHandler {
       constructor(
         @inject('findRoute') protected findRoute: FindRoute,
-        @inject('getFromContext') protected getFromContext: getFromContext,
+        @inject('getFromContext') protected getFromCntxt: getFromContext,
```

Comment:
IIRC, the property names should not be changed. Can we find out why they are shadowed?